solvation_analysis/tests/test_analysis_library.py

```diff
@@ -0,0 +1,66 @@
+import numpy as np
+import pytest
+
+from solvation_analysis.analysis_library import (
+    Speciation,
+    Coordination,
+    Pairing,
+)
+
+
+@pytest.mark.parametrize(
+    "cluster, percent",
```

Comment:
perhaps a note on how this test % data was generated.

Reply:
On how the bn-fec system was generated or specifically on how these percentages were generated? I used the method itself to calculate them... They do align with my previous knowledge of the system though.

Reply:
Ok no worries feel free to resolve.